modules/installation-vsphere-config-yaml.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere.adoc
+
+[id="installation-vsphere-config-yaml-{context}"]
+= Sample `install-config.yaml` file for vSphere
+
+You can customize the `install-config.yaml` file to specify more details about
+your {product-title} cluster's platform or modify the values of the required
+parameters.
+
+[source,yaml]
+----
+apiVersion: v1beta4
+baseDomain: example.com <1>
+compute:
+- name: worker
+  replicas: 0 <2>
+controlPlane:
+  name: master
+  replicas: 3
+metadata:
+  name: test <2>
+platform:
+  vsphere:
+    vcenter: your.vcenter.server <3>
+    username: username <4>
+    password: password <5>
+    datacenter: datacenter <6>
+    defaultDatastore: datastore <7>
+pullSecret: '{"auths": ...}' <8>
+sshKey: 'ssh-ed25519 AAAA...' <9>
+
+----
+<1> The base domain of the cluster. All DNS records must be sub-domains of this
+base and include the cluster name.
+<2> You must set the number of workers to `0`.
+<3> The cluster name.
+<4> The fully-qualified host name or IP address of the vCenter server.
+<5> The name of the user for accessing the server. This user must have at least
+the roles and privileges that are required for
+link:https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/vcp-roles.html[dynamic persistent volume provisioning]
+in vSphere.
+<6> The password associated with the vSphere user.
+<7> The vSphere datacenter.
+<8> The default vSphere datastore to use.
+<9> The pull secret that provides components in the cluster access to images for {product-title} components.
+<10> The default SSH key for the `core` user.
```

Comment:
It is not clear to me from reading this what the core user is. Is there documentation elsewhere that describes that each OpenShift machine has a core user that uses this SSH key? Also, why is it the "default" SSH key instead of "the" SSH key?